src/arrayMethodSort.js

```diff
@@ -4,9 +4,35 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    const comparator =
+      compareFunction ||
+      ((a, b) => {
+        const strA = String(a);
+        const strB = String(b);
+
+        return strA > strB ? 1 : -1;
```

Comment:
The comparator function should return 0 when `strA` is equal to `strB`. This is important for correctly handling cases where elements are equal.